Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 gem 'rails', '4.1.8'
 
 gem 'sqlite3'
-gem 'mysql'
+gem 'mysql2'
```

Comment:
I am also concerned about this gemfile change, currently al envs other than James' are using the mysql gem. @dshachtman ^^ what is the right gem?